packages/terra-consumer-nav/CHANGELOG.md

```diff
@@ -1,6 +1,19 @@
 ChangeLog
 =========
 
+# 0.1.0-BETA.3 - (--)
```

Comment:
can we make sure to add this date of whatever day you commit? Otherwise I have to keep updating it every time i release